### PR TITLE
Feature/search page

### PR DESCRIPTION
**Pull request by sattyframework**
Search index includes: product categories, services and news articles

Default search page 
image

Found Results with pagination 
image

### CODE REVIEW

File: resources/views/search/_results.antlers.html [hunk 1]
```diff
@@ -0,0 +1,33 @@
+<div>
+  {{ search:results index="default" as="results" }}
+        <ul class="pt-75 xl:pt-150">
+
+          {{ results }}
+
+            <li class="relative group pb-40 xl:pb-60">
+              <a {{ if taxonomy == 'product_categories' }} href="/products/{{ slug }}" {{ else }} href="{{url}}" {{ /if}}>
```
Comment (sattyframework) [inline resources/views/search/_results.antlers.html:8]: The taxonomy uses a custom route

File: resources/views/search/_results.antlers.html [hunk 2]
```diff
@@ -0,0 +1,35 @@
+<div>
```
Comment (FrameworkJames) [inline resources/views/search/_results.antlers.html:1]: This wrapping div seems unneeded.

File: resources/views/search/_results.antlers.html [hunk 3]
```diff
@@ -0,0 +1,35 @@
...
+                    <h3 class="heading">{{ title }}</h3>
+                  {{ /if }}
+
+                  <button class="btn-arrow btn-arrow--mustard ml-70 p-15">
+                    {{ svg src="arrow-right" class="group-hover:animate-arrow-slide" }}
+                  </button>
+                </div>
+
+                {{ if excerpt }}
+                  <p class="copy pt-30">{{excerpt}}</p>
```
Comment (FrameworkJames) [inline resources/views/search/_results.antlers.html:21]: I would line-clamp this excerpt.

File: resources/views/search/_results.antlers.html [hunk 4]
```diff
@@ -0,0 +1,35 @@
...
+
+            <li class="relative group pb-40 xl:pb-60">
+              <a {{ if taxonomy == 'product_categories' }} href="/products/{{ slug }}" {{ else }} href="{{url}}" {{ /if}}>
+
+                <div class="flex items-center">
+                  {{ if title }}
+                    <h3 class="heading">{{ title }}</h3>
+                  {{ /if }}
+
+                  <button class="btn-arrow btn-arrow--mustard ml-70 p-15">
```
Comment (FrameworkJames) [inline resources/views/search/_results.antlers.html:15]: You could keep the `a` tag here and use `btn-cover` to expand the link across the card.

**Review by FrameworkJames — CHANGES_REQUESTED** [review]

**Review by FrameworkJames — APPROVED** [review]

**Pull request merged**